packages/playground/src/components/execution/key-value-table.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHeader,
+  TableHead,
+  TableRow,
+  Text,
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger
+} from '@harnessio/canary'
+import { NavArrowDown } from '@harnessio/icons-noir'
+interface KeyValueTableProps {
+  tableTitleName: string
+  tableTitleVal: string
+  className?: string
+  tableVal: { name: string; value: string | object | [] }[]
```

Comment:
This is a loose type `[]`

Reply:
@jessieyu777 use these types instead

```
type KeyValuePair = {
  [key: string]: string | KeyValuePair;
};
```
and

`tableValue: KeyValuePair[]`

Reply:
```
const example: KeyValuePair[] = [
  { key1: "value1" },
  { key2: { nestedKey1: "nestedValue1", nestedKey2: { deepKey: "deepValue" } } },
];
```

Reply:
@jessieyu777 use this type instead.

```
type KeyValuePair = {
  name: string;
  value: string | KeyValuePair[];
};

const example: KeyValuePair[] = [
  { name: "item1", value: "value1" },
  { 
    name: "item2", 
    value: [
      { name: "nestedItem1", value: "nestedValue1" },
      { 
        name: "nestedItem2", 
        value: [
          { name: "deepItem", value: "deepValue" }
        ]
      }
    ] 
  }
];
```

Reply:
Cleaned